contracts/wrapper/NameWrapper.sol

```diff
@@ -302,6 +314,7 @@ contract NameWrapper is
             revert RenewalTooShort(bytes32(tokenId));
         }
         if (isWrapped(node)) {
+            // renew() protected by BaseRegistrar for non-owners, so owner should be correct owner
```

Comment:
What does this mean?

Reply:
They were just comments to note there was no way to accidentally set the owner to the previous owner when they don’t own it. Will remove the comments once approved